tests/test_context.py

```diff
@@ -65,7 +66,7 @@ class OuterContextComponent(component.Component):
     template_name = "simple_template.html"
 
     def get_context_data(self):
-        return self.outer_context
+        return self.outer_context.flatten()
```

Comment:
Would it be because `get_context_data()` outpus a dictionary, and outer_contex is a `Context`? I think there was a similar issue you solved before.

Reply:
Oh yeah, good point, it could be connected to how we [coerce to Context in render](https://github.com/JuroOravec/django-components/blob/4ffccdf5dd9ab7b0c5d34c29928ed45efe55db65/src/django_components/component.py#L270)

Reply:
Or maybe this [one](https://github.com/JuroOravec/django-components/blob/4ffccdf5dd9ab7b0c5d34c29928ed45efe55db65/src/django_components/component.py#L379) gets called somewhere in the process?